### PR TITLE
Run prepare for tagger

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
Please wait for @pkajaba's changes in tagger before merging. I will modify git hash accordingly afterwards.

**Comment by fridex** [comment]
OK, so let's proceed for now with this one, we can add Maven collector later.

File: Dockerfile [hunk 1]
```diff
@@ -43,3 +43,6 @@ RUN find ${HOME} -mindepth 1 -delete
 RUN mkdir -p /tmp/install_deps/patches/
 COPY hack/patches/* /tmp/install_deps/patches/
 RUN /tmp/install_deps/patches/apply_patches.sh
+
+# Install external resources needed for automated tagging
+RUN python3 -c 'import f8a_tagger; f8a_tagger.prepare()'
```
Comment (jpopelka) [inline Dockerfile]: Could we move this closer to the tagger installation, i.e. line 37 ?

**Comment by fridex** [comment]
Fixes: https://github.com/fabric8-analytics/fabric8-analytics-worker/issues/270

File: Dockerfile (hunk 1 above)
Comment (fridex) [reply Dockerfile]: Sure, will do.

Comment (fridex) [reply Dockerfile]: Or we could add this directly to the install script.

**Pull request merged**